packages/phasor/src/utils/point-location.ts

```diff
@@ -0,0 +1,28 @@
+import { type IVec } from './vec.js';
+
+export class PointLocation extends Array<number> implements IVec {
```

Comment:
Not sure if this is always used as marking relative position? If true, this can be better named as `RelativeLocation`.

Reply:
the point in PointLocation is stiil canvas point, it's not related to something.
But it brings the tangent if the point is on outline of some surfaceElements.

and maybe in the future, it will bring more infomation.